plugin/doge.vim

```
@@ -1,11 +1,20 @@
 let s:save_cpo = &cpoptions
 set cpoptions&vim
 
-let s:unsupported_vim = !has('nvim') && (v:version < 800 && !has('patch-7.4.2219'))
+let s:unsupported_vim = !has('nvim') && (v:version < 700 && !has('patch-7.4.2219'))
```

What was wrong about this check? I thought vim could be version 700 but not have this patch.

I tested it in vim 7.4.160 and both `has('patch-7.4.2219')` and `has('patch2219')` return 0, the second one should be wrong anyway because it returns 0 also in later vims, the first one returns 1 also in neovim.

---

`has('patch-7.4.2219')` will return 1 in Vim 8+ as well. The `has('patch2219')` will check in your current major version if it contains the 2219 patch. If I am on Vim8 it will check for 8.x.2219 and that's not something we want. If you are below 7 or you do not have 7.4.2219 then we can throw an error. 

For NeoVim we use `has('nvim')` as an additional check.

So I will adjust the statement. It should be good after I replace the `&&` with `||`.

---

`has('patch-7.4.2119')` will return 1 in Vim 8+ as well. The `has('patch2119')` will check in your current major version if it contains the 2119 patch, if I am on Vim 7 it will check if I have 7.x.2119, if I am on Vim8 it will check for 8.x.2119 and that's not something we want. If you are below 7 OR you do not have 7.4.2119 then we can throw an error. 

For NeoVim we use `has('nvim')` as an additional check.

What I just did was update a typo in the version (it should've been `2119` instead of `2219` and I removed the `v:version` check, since this version is already checked now using `has('patch-7.4.2119')`. If you use Vim 8 this will still return `1`. If you are below this version you should get a `0` returned. Can you confirm this?

---

As of https://github.com/kkoomen/vim-doge/pull/42/commits/14e4ce5cb1f764934b53ae532e0bc4f8567727ee, it should be ok.